### ISSUE TITLE
vuetify/no-deprecated-props incorrectly reports v-snackbar text prop as removed (Vuetify ≥ 3.4)

### ISSUE THREAD

**Issue by abab5601** [vuetifyjs/eslint-plugin-vuetify#129]
### Description

`eslint-plugin-vuetify` rule `vuetify/no-deprecated-props` incorrectly reports that the `text` prop on `v-snackbar` has been removed.

In my project, `v-snackbar` still supports the `text` prop, but ESLint shows a warning:

```text
'text' has been removed eslint(vuetify/no-deprecated-props)
```

This appears to be a false positive.

### Environment

- vuetify: 4.0.2
- eslint-plugin-vuetify: 2.7.0
- eslint: 10.0.3
- node: 22.21.1
- pnpm: 10.18.2
- nuxt: 4.1.3
- vue: 3.5.30

### Code Example

```vue
<v-snackbar text="表單已完成帳號建立，請從定義使用者進行後續操作">
  <template #activator="{ props: activatorProps }">
    <v-btn
      v-bind="activatorProps"
      variant="text"
      color="primary"
      icon="mdi-pencil"
      density="comfortable"
    />
  </template>
</v-snackbar>
```

### Current Behavior

ESLint reports:

```text
'text' has been removed eslint(vuetify/no-deprecated-props)
```

### Expected Behavior

No ESLint warning should appear, because `text` is a valid prop for `v-snackbar` in the current Vuetify version I am using.

### Related Packages

```json
{
  "dependencies": {
    "eslint": "^10.0.3",
    "nuxt": "^4.1.3",
    "vue": "^3.5.30",
    "vuetify": "^4.0.2"
  },
  "devDependencies": {
    "eslint-plugin-vuetify": "2.7.0"
  },
  "packageManager": "pnpm@10.18.2"
}
```

### Additional Notes

This looks like the rule metadata for `v-snackbar` may be outdated, and still treats `text` as a removed prop.

It would be helpful if the rule could be updated to match the current Vuetify API.
<img width="1041" height="256" alt="Image" src="https://github.com/user-attachments/assets/964034d1-bed9-4c47-889f-7120ac486d17" />
